src/include/wiredtiger.in

```diff
@@ -3822,9 +3812,10 @@ struct __wt_file_system {
 	 * @param file_system the WT_FILE_SYSTEM
 	 * @param session the current WiredTiger session
 	 * @param name the name of the file system object
+	 * @param durable if the operation requires durability
 	 */
-	int (*fs_remove)(
-	    WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name);
+	int (*fs_remove)(WT_FILE_SYSTEM *file_system,
+	    WT_SESSION *session, const char *name, bool durable);
```

Comment:
I prefer flags over a boolean. The API is only months old and we've found a reason to add a flag, which makes me think we'll probably want more in the future. Using flags increases our documentation burden, but gives us more flexibility to make enhancements without API changes in the future.

I *think* I prefer reusing ``WT_FS_OPEN_DURABLE`` to having two durable flags in the API, but that introduces constraints on flags ranges for future flags.